tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -456,28 +457,36 @@ public function makeDavRequest(
 	 * @param string|null $height
 	 *
 	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
 	 */
 	public function downloadPreviews(string $user, ?string $path, ?string $doDavRequestAsUser, ?string $width, ?string $height):void {
-		$user = $this->getActualUsername($user);
-		$doDavRequestAsUser = $this->getActualUsername($doDavRequestAsUser);
 		$urlParameter = [
 			'x' => $width,
 			'y' => $height,
-			'forceIcon' => '0',
 			'preview' => '1'
 		];
-		$this->response = $this->makeDavRequest(
+		$url = $this->getBaseUrl() . '/remote.php';
+		$davVersion = $this->getDavPathVersion();
+		if ($davVersion === WebDavHelper::DAV_VERSION_SPACES) {
+			$fileId = urlencode(strstr($this->getFileIdForPath($doDavRequestAsUser ?? $user, $path), '!', true));
+			$url .= '/dav/spaces/' . $fileId . '/';
```

Comment:
should we use resource-id or resource-name? I would vote for `resource-name`
`/dav/spaces/<personal-space-id>/$path`

Reply:
cc @ScharfViktor